regression/goto-analyzer/command_line_01/test.desc

```diff
@@ -0,0 +1,12 @@
+CORE
+main.c
+--verify --recursive-interprocedural --ahistorical --constants --one-domain-per-history
+\[main.assertion.1\] line 13 assertion v != 0: FAILURE \(if reachable\)
+\[f00.assertion.1\] line 5 assertion x != 0: FAILURE \(if reachable\)
+Summary: 0 pass, 2 fail if reachable, 0 unknown
+^EXIT=0$
```

Comment:
Just a question: `goto-analyzer` returns `0` even when there are failing assertions?

Reply:
Yeah... At the moment it does because the analysis has run successfully / it hasn't shown the presence of bugs.  I can see an argument for it returning 10 but I think that is beyond the scope of the current PR.